pkg/settings/setting.go

```diff
@@ -20,6 +20,8 @@ var (
 	InjectDefaults string
 
 	AgentImage                        = NewSetting("agent-image", "rancher/rancher-agent:master-head")
+	AgentRolloutTimeout               = NewSetting("agent-rollout-timeout", "300s")
+	AgentRolloutWait                  = NewSetting("agent-rollout-wait", "true")
```

Comment:
This is changing existing behavior and might have side effects so we need docs on this so users know what they can configure if something happens.

Reply:
Yes, this is changing the cluster update behaviour as mentioned at previous comments. Agreed that might have side effects, so we should deep test it. This was one of the main reasons to add the parameters, to disable/config the "new" behaviour if needed.